### PR TITLE
Add editer awesome

### PR DESCRIPTION
**Pull request by kingyueyang**

### CODE REVIEW

**Comment by emijrp** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -81,6 +82,11 @@ A curated list of awesome curated lists of many topics.
 
 * [Search Engine Optimization (SEO)](https://github.com/marcobiedermann/search-engine-optimization) - A helpful checklist / collection of Search Engine Optimization (SEO) tips and technics.
 
+## Editer
```
Comment (waldyrious) [inline README.md]: Shouldn't this be "Editors"?

Comment (emijrp) [inline README.md]: Yes, fixing...

2015-11-21 19:52 GMT+01:00 Waldir Pimenta <notifications@github.com>:

**Comment by emijrp** [comment]
@waldyrious Fixed 6c40a79e2d8fa6fd3947353acc5722acf938e7fd

**Comment by waldyrious** [comment]
:+1:

**Comment by kingyueyang** [comment]
thx @waldyrious

**Pull request merged**